javascript-timers/interval.js

```diff
@@ -0,0 +1,13 @@
+function countdown() {
+  const $countdownDisplay = document.querySelector('.countdown-display');
```

Comment:
This should be done globally. For the most part, `querySelector`s should always be done globally. By including it inside a function, you're forcing us to traverse the DOM over and over every single time this function gets called